crates/mamoru-sui-sniffer/src/lib.rs

```diff
@@ -80,7 +80,7 @@ impl SuiSniffer {
 
             let before_ms = Utc::now().timestamp_millis();
 
-            register_call_traces(ctx_builder.data_mut(), seq, call_traces);
+            register_call_traces(ctx_builder.data_mut(), seq, call_traces.clone());
```

Comment:
Don't do clones, it takes too much memory.

Reply:
fixed